api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -138,6 +138,11 @@ type AgentClusterInstallSpec struct {
 	// IngressVIP is the virtual IP used for cluster ingress traffic.
 	// +optional
 	IngressVIP string `json:"ingressVIP,omitempty"`
+
+	// HoldInstallation if true it will prevent installation to start automatically.
```

Comment:
when an agent gets approved we have:
`Approved = True`

with `HoldInstallation` it kinda the opposite, where `True` means 'not approved'
for consistency, maybe we should have `ApproveInstallation` instead?

Reply:
We had this discussion with UX and architects, we prefer the default to be automatic. 
We can add defaults with webhooks but it's breaking the API for existing clusters/ZTP flows.

Reply:
ok